docs/atlascli/command/atlas-backups-restores-list.txt

```diff
@@ -85,3 +85,11 @@ Inherited Options
      - false
      - Profile to use from your configuration file.
 
+Examples
+--------
+
+.. code-block::
+
+ The following example retrieves the continuous backup restore jobs for the cluster Cluster0::
```

Comment:
Oddly, the second ":" was not showing up before so now there's doubles... sorry to make you change twice. Please remove one
```suggestion
 The following example retrieves the continuous backup restore jobs for the cluster Cluster0:
```